controllers/withwatcher/kyma_controller_test.go

```diff
@@ -21,90 +27,212 @@ import (
 )
 
 const (
-	webhookConfigNameTpl           = "%s-webhook"
 	servicePathTpl                 = "/validate/%s"
 	specSubresources               = "*"
 	statusSubresources             = "*/status"
 	expectedWebhookNamePartsLength = 4
+	dummyCaBundleData              = "amVsbHlmaXNoLXdhcy1oZXJl"
 )
 
-var _ = Describe("Kyma with multiple module CRs in remote sync mode", Ordered, func() {
-	kyma := NewTestKyma("kyma-remote-sync")
-	watcherCrForKyma := createWatcherCR("skr-webhook-manager", true)
+var (
+	ErrExpectedAtLeastOneWebhook       = errors.New("expected at least one webhook configured")
+	ErrWebhookConfigForWatcherNotFound = errors.New("webhook config matching Watcher CR not found")
+	ErrWebhookNamePartsNumberMismatch  = errors.New("webhook name dot separated parts number mismatch")
+	ErrManagedByLabelNotFound          = errors.New("managed-by label not found")
+	ErrModuleNameMismatch              = errors.New("module name mismatch")
+	ErrSvcPathMismatch                 = errors.New("service path mismatch")
+	ErrWatchLabelsMismatch             = errors.New("watch labels mismatch")
+	ErrStatusSubResourcesMismatch      = errors.New("status sub-resources mismatch")
+	ErrSpecSubResourcesMismatch        = errors.New("spec sub-resources mismatch")
+)
 
+var _ = Describe("Kyma with multiple module CRs in remote sync mode", Ordered, Serial, func() {
```

Comment:
Ordered - I thought we decided to get rid of them?
Is setting Serial by mistake?